packages/block-library/src/heading/heading-toolbar.js

```diff
@@ -25,7 +25,9 @@ class HeadingToolbar extends Component {
 	render() {
 		const { minLevel, maxLevel, selectedLevel, onChange } = this.props;
 		return (
-			<Toolbar controls={ range( minLevel, maxLevel ).map( ( index ) => this.createLevelControl( index, selectedLevel, onChange ) ) } />
+			<>
```

Comment:
There is no longer need to use `Fragment` (`<>`).

Reply:
thank you missed that!